### PR TITLE
IC5 role definition: define IC5 through profiles

### PR DESCRIPTION
**Pull request by lguychard**
See [context on Slack](https://sourcegraph.slack.com/archives/C021VSRSGUF/p1641241683034700).

This PR:

- Documents that the IC5 level is defined through IC5 profiles, to acknowledge that impact beyond IC4 varies between individuals.
- Documents common traits of all IC5 profiles.
- Documents a process for adding a new IC5 profile, ensuring that new IC5 position are fairly calibrated against existing ones.

The following **non-goals** are explicitly and intentionally not tackled:

- Adding actual IC5 profiles: this would be an immediate next step after merging this.
- Deciding whether we call IC5 "Staff": this can be tackled as a follow-up, what is important here is defining career progression beyond IC4.
- Defining IC6 and 7.

### CODE REVIEW

**Comment by lguychard** [comment]
@nicksnyder can you please give this an initial review to make sure it is aligned with your expectations following our discussion before the holiday? Thanks!

File: content/departments/product-engineering/engineering/career-development/ic5/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# IC5 roles at Sourcegraph
+
+This page is a list of profiles for IC5 roles at Sourcegraph. The IC5 level is defined through these profiles, acknowledging that impact at IC5 and beyond varies between individuals. As they form the level definition, these profiles can be used in career growth conversations between ICs and EMs, in [calibration](../talent-review-process.md#calibration-meeting) discussions between EMs and directors.
```
Comment (jeanduplessis) [inline content/departments/product-engineering/engineering/career-development/ic5/index.md]:
```suggestion
This page is a list of profiles for IC5 roles at Sourcegraph. The IC5 level is defined through these profiles, acknowledging that impact at IC5 and beyond varies between individuals. As they form the level definition, these profiles can be used in career growth conversations between ICs and EMs, and in [calibration](../talent-review-process.md#calibration-meeting) discussions between EMs and directors.
```

**Review by nicksnyder — APPROVED** [review]
lgtm, thank you!

**Comment by lguychard** [comment]
Directors (@yinquanteo, @jeanduplessis, @bcreager), can you please take a look? See [context here](https://sourcegraph.slack.com/archives/C021VSRSGUF/p1641241683034700). Thank you!

**Review by jeanduplessis — APPROVED** [review]

File: content/departments/product-engineering/engineering/career-development/ic5/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+We recommend <a href="https://drive.google.com/drive/folders/1fnw5gmlLDDZb9wDodY3KfcTaM3OOQL7G" target="_blank" rel="noopener noreferrer">our company’s copy of “Staff Engineer” by Will Larson</a> as reading material.
+
+## Common traits of IC5 profiles
+
+While the realization of the IC5 roles in practice will vary, a common starting point is always a **business need** for the role: in some cases the business may need an IC to tackle broad, cross-cutting technical problems at the org or company level. In others, it may need a deep technical expert for a specific area, capable of devising solutions to highly complex and specialized technical problems.
+
+The business need may evolve over time, and IC5s are expected to adjust their focus accordingly: the expectation to proactively and continuously **identify and advocate for work that will bring strategic value** is another common trait of all IC5 positions.
```
Comment (yinquanteo) [inline content/departments/product-engineering/engineering/career-development/ic5/index.md:11]: This is a great initial list. Some other potentially common traits (we don't have to include as part of this change but I wanted to share)
- Leading others: 
  - Ability to bring others along (convincing/influence) even in situations where there are strong opposing views on matters of organizational significance
  - Something around effective delegation but being overall accountable for the area they oversee?
- Time horizon that they're operating at. This should be higher than the expectations we have of our IC4s.

**Review by yinquanteo — APPROVED** [review]
Lgtm! Added some thoughts on potentially common IC5 traits

**Comment by lguychard** [comment]
Merging as-is since this has a few approvals, and reflects how we want to operate with IC5 better than what is currently in the handbook. We'll of course iterate on the content as needed.

**Pull request merged**